backend/schema/validate.go

```diff
@@ -32,6 +32,18 @@ var (
 	}
 )
 
+type dependencyVertex struct {
```

Comment:
Let's put the types/consts down near the function, it makes it easier to understand everything at once.